src/Faker/Provider/es_CO/Person.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Faker\Provider\es_CO;
+
+class Person extends \Faker\Provider\Person
+{
+    /**
+     * @link https://culturalatlas.sbs.com.au/colombian-culture/naming-4ad0be20-a64e-4505-aaa1-e4ade4c7ba05#naming-4ad0be20-a64e-4505-aaa1-e4ade4c7ba05
+     */
+    protected static $maleNameFormats = array(
+        '{{firstNameMale}} {{lastName}} {{lastName}}',
+        '{{firstNameMale}} de {{lastName}} {{lastName}}',
+        '{{firstNameMale}} {{firstNameMale}} {{lastName}} {{lastName}}',
+        '{{firstNameMale}} {{preposition}} {{firstNameMale}} {{lastName}} {{lastName}}',
+        '{{firstNameMale}} {{firstNameMale}} {{preposition}} {{firstNameMale}} {{lastName}} {{lastName}}',
+    );
+
+    /**
+     * @link https://culturalatlas.sbs.com.au/colombian-culture/naming-4ad0be20-a64e-4505-aaa1-e4ade4c7ba05#naming-4ad0be20-a64e-4505-aaa1-e4ade4c7ba05
+     */
+    protected static $femaleNameFormats = array(
+        '{{firstNameFemale}} {{lastName}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}} {{lastName}}',
+        '{{firstNameFemale}} de {{lastName}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}} {{lastName}} de {{lastName}}',
+        '{{firstNameFemale}} {{firstNameFemale}} {{lastName}} {{lastName}}',
+        '{{firstNameFemale}} {{preposition}} {{firstNameFemale}} {{lastName}} {{lastName}}',
+        '{{firstNameFemale}} {{firstNameFemale}} {{lastName}} {{lastName}} de {{lastName}}',
+        '{{firstNameFemale}} {{preposition}} {{firstNameFemale}} {{lastName}} {{lastName}} de {{lastName}}',
+    );
+
+    /**
+     * @link https://www.elespectador.com/noticias/politica/estos-fueron-los-nombres-mas-populares-del-2018-en-colombia-segun-la-registraduria-articulo-828629
```

Comment:
This content is copyrighted, you can't copy it in an MIT licensed library. Please find another data source. Same for other name arrays.